_posts/0000-01-02-pietromacalossi.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
Check this page of classical memes [facebook classicalartmemes](https://www.facebook.com/classicalartmemes/)